src/redis_cache.cc

```diff
@@ -89,6 +89,15 @@ setOption(
   }
 }
 
+TRITONSERVER_Error*
+handleError(
+    const std::string& message, const std::string& key,
+    const std::string& cause)
+{
+  std::ostringstream msg;
+  msg << message << key << " from cache. " << cause;
```

Comment:
Wording will be a bit awkward

"failed to insert key: keyname from cache" 

maybe have a flag in here to adjust the wording slightly depending on if it's an insert/lookup

Reply:
I just removed "from cache". Mentions of cache will be included in wrapped Triton errors. I think "failed to insert key" and "failed to retrieve key" are descriptive enough for now.